green_cli/authenticators/jade.py

```diff
@@ -348,14 +348,20 @@ def _map_input(input: Dict) -> Dict:
             }
 
             # If non-trivial tx, also send any blinding information per input
-            if txtype != tx.TxType.SEND_PAYMENT and 'assetblinder' in input:
-                mapped['abf'] = bytes.fromhex(input['assetblinder'])[::-1]
+            if txtype != tx.TxType.SEND_PAYMENT and input.get('is_blinded'):
                 mapped['asset_id'] = bytes.fromhex(input['asset_id'])
                 mapped['asset_generator'] = bytes.fromhex(input['asset_tag'])
+
+                if 'assetblinder' in input:
```

Comment:
Its possible albeit potentially inefficient to provide both the blinder and the blind proofs. If the proofs are present they should be verified IMO even if the blinder is provided.

Reply:
Note this kind of thing can happen e.g. when callers augment their own PSETs for signing then strip out the sigs to poke into a PSET they are sending someone else.